CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 ##### Fixes
 
+* [7926](https://github.com/grafana/loki/pull/7926) **MichelHollands**: Fix validation for pattern and rexexp parsers to they don't panic in query-frontend.
```

Comment:
The panic in the query-frontend is to identify these sort of problems, that otherwise would be missed. Therefore, I think we can just mention the fix of the parser

```suggestion
* [7926](https://github.com/grafana/loki/pull/7926) **MichelHollands**: Fix validation for pattern and regexp parsers.
```